### ISSUE TITLE
Yolact Module

### ISSUE THREAD

**Issue by haraldger** [haraldger/Real-Time-Instance-Segmentation#7]
Implement Yolact module. This is the full network, incorporating all the sub-modules previously inmplemented: backbone, fpn, prediction heads and protonet. 

The branch requires:
- PyTorch Module with all components
- Forward pass
- Loss function and calculation
- Fast NMS implementation
- Assembling of masks, cropping and thresholding
